LayoutTests/pdf/unifiedpdf/images/jpeg2000-embed-pdf.html

```diff
@@ -0,0 +1,42 @@
+<!-- webkit-test-runner [ PDFPluginHUDEnabled=false ] -->
+<style>
+    .container {
+        position: absolute;
+        width: 200px;
+        height: 200px;
+        overflow: clip;
+        cursor: none;
+    }
+    embed {
+        width: 400px;
+        height: 400px;
+        transform: translate(-100px, -100px);
+        cursor: none;
+    }
+</style>
+<body>
+    <div class="container">
+        <embed src="../resources/400x400-green-jpeg2000.pdf">
+    </div>
+    <script>
+        if (window.testRunner)
+            testRunner.waitUntilDone();
+
+
+        addEventListener("load", async () => {
+            if (!window.testRunner)
+                return;
+
+            let rafs = 0;
+
+            let countRafs = (timestamp) => {
+                if (++rafs < 50)
```

Comment:
If you enable UnifiedPDF via the header you should be able to avoid waiting for the PDF fade in animation.

Reply:
Adding 
```
<!-- webkit-test-runner [ UnifiedPDFEnabled=true ] -->
```
did not prevent the PDF fade animation.